normits_demand/models/notem/production_models.py

```diff
@@ -321,7 +322,11 @@ def run(self,
             #  Output some audits of what demand was before and after control
             #  By segment.
             if self.constraint_paths is not None:
-                self._logger.info("No code implemented to constrain productions." )
+                msg=(
+                    "No code implemented to constrain productions."
+                )
```

Comment:
This looks like it would fit on one line. Can't be any longer than the line above